concent_api/core/payments/mock.py

```diff
@@ -8,7 +9,7 @@ def get_client_eth_account():
     return True
 
 
-def get_number_of_eth_block(request = None):  # pylint: disable=inconsistent-return-statements
+def get_number_of_eth_block(request = None, _subtask_results_accepted_list = None, _list_of_transactions = None, _list_of_forced_payments = None):  # pylint: disable=inconsistent-return-statements
```

Comment:
Please also change in `payments\base.py`. I see there are now two function calling `get_number_of_eth_block`. There should be only one, and second one should call `payment_summary()`. Please also make sure that both functions signature is correct in base and mock backends, and their usage in other code is correct.

Reply:
Okey, I see. Function `get_number_of_eth_block` is currently not used at all in our code, and `payment_summary` is used only once. Should I maybe remove `get_number_of_eth_block` completely for right now?

Reply:
About removal, please discuss it with @kbeker .

This it not fixed yet:
1. `payment_summary` in base.py should call backend.payment_summary
2. `get_number_of_eth_block` in base.py should have only `backend, request` parameters.
3. `get_number_of_eth_block` in mock.py should have only `request` parameter.

Reply:
Done, and I'll talk to @kbeker when I'll got a chance :)

Reply:
Ok, thanks.